tests/test_samplers.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
```

Comment:
@mberr can we delete this now? were there any ideas here that you wanted for the pseudotyped negative sampler tests?

Reply:
I guess we can delete if, since everything here is also in the updated tests in #447

Reply:
We can add some pseudo-type tests though 🙂

Reply:
Yes I think it is

Reply:
6823af2 - it currently fails :sweat_smile: